### DISCUSSION TITLE
Problema com botões

### DISCUSSION THREAD

**Discussion by maicolgenz** [salman0ansari/whatsapp-api-nodejs#701] [General]
No envio de botões funciona bem, mas em muitos aparelhos não aparece, apenas no webwhatsapp.
Já verifiquei em um aparelho, e está na última versão do whatsapp.
O que poderia ser, algo que eu fiz ou é bug da api mesmo?

Testes com a api da meta funciona normal para o mesmo telefone.

**Comment by lpinheirojunior** [comment]
Essa é uma característica dessa api. Alguns aparelhos mais antigos, não exibem os botões.
